cli/cmd/configure_domain.go

```diff
@@ -274,7 +266,8 @@ func updateKeptnAPIIngress(path, domain string) error {
 	}
 
 	if err := file.Replace(keptnAPIIngress,
-		file.PlaceholderReplacement{PlaceholderValue: "domain.placeholder", DesiredValue: domain}); err != nil {
+		file.PlaceholderReplacement{PlaceholderValue: "domain.placeholder", DesiredValue: domain},
+		file.PlaceholderReplacement{PlaceholderValue: "ingress.placeholder", DesiredValue: "nginx"}); err != nil {
 		return err
```

Comment:
Please add inline documentation about what the above 4 lines do (e.g., replace xyz in file foo.bar to provide something)

Reply:
Done.